app/src/main/java/no/usn/mob3000/ui/screens/chess/train/group/CreateGroupScreen.kt

```diff
@@ -54,8 +55,8 @@ fun CreateGroupScreen(
             modifier = Modifier.padding(innerPadding),
             bottomBar = {
                 BottomAppBar(
-                    containerColor = MaterialTheme.colorScheme.surface,
-                    contentColor = MaterialTheme.colorScheme.onSurface,
+                    containerColor = MaterialTheme.colorScheme.primary,
+                    contentColor = MaterialTheme.colorScheme.onPrimary,
```

Comment:
These parameters should keep using `surface` and `onSurface`. The same `Color.kt` values can simply be applied to those parameters in the color schemes and then made use of.